pkg/instrumentation/helper.go

```diff
@@ -14,7 +14,23 @@
 
 package instrumentation
 
-import corev1 "k8s.io/api/core/v1"
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+func defaultInitContainerResourceRequirements() corev1.ResourceRequirements {
+	return corev1.ResourceRequirements{
+		Limits: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("500m"),
+			corev1.ResourceMemory: resource.MustParse("128Mi"),
```

Comment:
Similar to the other comments in the PR... these should definitely be configurable on top of these defaults as part of the Instrumentation CRD or the annotations to set auto-instrumentation.